frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/vm-settings-tab/provision-source.tsx

```diff
@@ -105,7 +105,7 @@ export const ProvisionSourceComponent: React.FC<ProvisionSourceComponentProps> =
   ({ provisionSourceField, onChange, goToStorageStep, goToNetworkingStep }) => {
     const { t } = useTranslation();
     const provisionSourceValue = iGetFieldValue(provisionSourceField);
-    const sources: string[] = iGet(provisionSourceField, 'sources');
+    const sources: string[] = [...iGet(provisionSourceField, 'sources')];
```

Comment:
I think iGet(provisionSourceField, 'sources').toJS() can also work here as far as I tested, but it doesn't really matter. whatever you prefer.